crates/swc_plugin/src/allocation.rs

```diff
@@ -31,5 +33,46 @@ pub extern "C" fn __alloc(size: i32) -> *mut u8 {
 pub extern "C" fn __free(ptr: *mut u8, size: i32) -> i32 {
```

Comment:
Did you leave this function intentionally?

Reply:
These are actual function being used currently. The new implementation is guarded via unused flag (https://github.com/swc-project/swc/pull/7197/files/8ab88722c78c828a2bfa3990cf2b23a185830e1e#diff-ae1999f472ec8ffb61b34fb95390e186e01553af65a00ab8b02f75433ea5f182R44).

I would like to replace, but having it in this PR would not be a good idea since it'll make bisect hard if something starts to breaking.

Reply:
Thank you!